.code-samples.meilisearch.yaml

```
@@ -577,4 +655,4 @@ document_guide_add_document_primary_key: |-
         "poster": "https://image.tmdb.org/t/p/w1280/xnopI5Xtky18MPhK40cZAGAOVeV.jpg",
         "overview": "A boy is given the ability to become an adult superhero in times of need with a single magic word.",
         "release_date": "2019-03-23"
-     }]' 
+     }]' 
```

Not a big issue, but useless addition here in this PR. We would rather avoid useless changes in PRs since they can "spoil" a future PR. This is for a better readability :)

---

Apologies, I'm having a hard time letting go of old habits. Most JS style guides I used recommended never leaving newlines at the end of files.

```suggestion
     }]' 

```